app/src/main/java/com/framgia/wsm/screen/login/LoginModule.java

```diff
@@ -4,11 +4,15 @@
 import android.content.Context;
 import android.support.annotation.NonNull;
 import com.framgia.wsm.data.source.UserRepository;
+import com.framgia.wsm.data.source.local.UserLocalDataSource;
+import com.framgia.wsm.data.source.local.sharedprf.SharedPrefsApi;
```

Comment:
ERROR: Unused import: com.framgia.wsm.data.source.local.sharedprf.SharedPrefsApi. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports